source/style-guide/includes/terms/t.rst

```diff
@@ -140,11 +140,20 @@ T
      *timeout* as one word when using the term as a noun or adjective.
 
    timestamp
-     Use *timestamp* as one word. Use only as a noun.
+     Use *timestamp* as one word. Use only as a noun. Use *timestamp* 
+     instead of *date* wherever applicable to avoid confusion.
+
+     .. seealso:: 
+       
+        :term:`date`
 
    time zone
      Use *time zone* as two words.
 
+   toggle
```

Comment:
[Question]

Should we specify that this definition is referring to toggle as a noun?